test/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationServiceTest.scala

```diff
@@ -15,29 +16,34 @@ class OAuth2TokenValidationServiceTest extends BaseSpec with Matchers with Route
   val mockTokenValidationService = ValidationTestRegistry.tokenValidationService
   val realValidationService = ValidationServiceRegistry.tokenValidationService
 
-  "Validation service" should "reject dummy token after validation remote server" in {
+  "Validation service" must "reject dummy token after validation remote server" in {
     whenReady(mockTokenValidationService.validate(Some(wrongToken))) { either =>
-      either should be('left)
+      either must be('left)
```

Comment:
99% of all scala test code written across all our projects use "must"-style matching. We even once had a document about such and other conventions. It makes no difference other than making it easier for new developer to dive into different projects and know the syntax.